dask/distributed/__init__.py

```diff
@@ -2,3 +2,10 @@
 from .scheduler import Scheduler
 from .client import Client
 from .ipython_utils import dask_client_from_ipclient
+
+from warnings import warn
+
+warn(FutureWarning('''
```

Comment:
That makes sense.  Apparently DeprecationWarnings are silenced by default.
Not sure if that changes what's best.

On Fri, Oct 16, 2015 at 9:15 AM, Jim Crist <notifications@github.com> wrote:

> In dask/distributed/__init__.py
> <https://github.com/blaze/dask/pull/786#discussion_r42260399>:
>
> > @@ -2,3 +2,10 @@
> >  from .scheduler import Scheduler
> >  from .client import Client
> >  from .ipython_utils import dask_client_from_ipclient
> > +
> > +from warnings import warn
> > +
> > +warn(FutureWarning('''
>
> Should probably be a DeprecationWarning instead. FutureWarning is about
> future semantic changes, not deprecated modules.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/blaze/dask/pull/786/files#r42260399>.
>